src/main/java/io/cdap/plugin/google/common/GoogleDriveClient.java

```diff
@@ -98,7 +105,7 @@ protected Credential getCredentials() throws IOException {
   }
 
   protected List<String> getRequiredScopes() {
-    return Collections.singletonList(DriveScopes.DRIVE_READONLY);
+    return Collections.singletonList(DriveScopes.DRIVE);
```

Comment:
why did we need to make this change?

Reply:
This was also as part of the testings that we have done. We were dealing with this error message when testing the pipeline:

`404 Not Found { "code" : 404, "errors" : [ { "domain" : "global", "location" : "fileId", "locationType" : "parameter", "message" : "File not found: 1ZqAr0q6gITvjqvfOJqkSIxd-usTNhvJy.", "reason" : "notFound" } ], "message" : "File not found: 1ZqAr0q6gITvjqvfOJqkSIxd-usTNhvJy." }. Please check the system logs for more details.
`

...and checking the documentation of Google Drive API for 404 error type, stating that:

To fix this error:

```
1. Inform the user they don't have read access to the file or that the file doesn't exist.
2. Instruct the user to ask the owner for permission to the file.
```
...we changed the SCOPE from `DRIVE_READONLY` to `DRIVE`.

But afterwards, when we verified that in fact we were not in the right path when testing the pipelines and DRIVE_READONLY was enough for read/write, we changed back to its initial SCOPE. 

In the last commit, we reverted back the scope to `DRIVE_READONLY`.

Reply:
With SCOPE set to `DRIVE` does everything work correctly?

Reply:
Actually, everything also works correctly with SCOPE set to `DRIVE_READONLY`, that's why we reverted back the code.